spec/rubocop/cop/rspec/factory_bot/consistent_parentheses_style_spec.rb

```diff
@@ -275,5 +275,25 @@
         end
       RUBY
     end
+
+    context 'when create and first argument are not on same line' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY)
+          create(
+            :user
+          )
+        RUBY
+      end
+    end
+
+    context 'when build and first argument are not on same line' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY)
+          build(
+            :user
```

Comment:
I think `build` is sufficiently covered. Instead, do you mind adding:
 - an example with two arguments (and a comma that is causing invalid syntax)
 - an example where the first argument and its comma are on the same line with the opening parenthesis